Add AuctionHouse generated code

mike-metaplex opened the pull request.

ajamaica commented:
Remove the [.DS_Store](https://github.com/metaplex-foundation/metaplex-swift-program-library/pull/1/files#diff-8c82015a38ab7a2cb11aa604c9ad81f3f6e9424439620d7ba7e1073b1accdb5d)

mike-metaplex committed:
Add AuctionHouse generated code

mike-metaplex commented:
> Remove the [.DS_Store](https://github.com/metaplex-foundation/metaplex-swift-program-library/pull/1/files#diff-8c82015a38ab7a2cb11aa604c9ad81f3f6e9424439620d7ba7e1073b1accdb5d)

Removed and added to `.gitignore`
